src/modules/cpu/host_tensor_augmentations.hpp

```diff
@@ -1065,4 +1065,1908 @@ RppStatus brightness_i8_i8_host_tensor(Rpp8s *srcPtr,
     return RPP_SUCCESS;
 }
 
+/************ gamma_correction ************/
+
+RppStatus gamma_correction_u8_u8_host_tensor(Rpp8u *srcPtr,
+                                             RpptDescPtr srcDescPtr,
+                                             Rpp8u *dstPtr,
+                                             RpptDescPtr dstDescPtr,
+                                             Rpp32f *gammaTensor,
+                                             RpptROIPtr roiTensorPtrSrc,
+                                             RpptRoiType roiType,
+                                             RppLayoutParams layoutParams)
+{
```

Comment:
I see a lot of empty lines in the code. Use empty lines only where needed to make code readable. Too many is not preferred.

Reply:
Done